src/Sylius/Bundle/TaxonomyBundle/Controller/TaxonPositionController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\TaxonomyBundle\Controller;
+
+use Doctrine\Persistence\ObjectManager;
+use Sylius\Component\Taxonomy\Model\TaxonInterface;
+use Sylius\Component\Taxonomy\Repository\TaxonRepositoryInterface;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Webmozart\Assert\Assert;
+
+final class TaxonPositionController
+{
+    public function __construct(
+        private TaxonRepositoryInterface $taxonRepository,
+        private ObjectManager $taxonManager,
+    ) {
+    }
+
+    public function moveUpAction(int $id): Response
+    {
+        /** @var TaxonInterface|null $taxonToBeMoved */
+        $taxonToBeMoved = $this->taxonRepository->find($id);
```

Comment:
It would be good to handle not found error in similar way as it's done at [AdminBundle](https://github.com/Sylius/Sylius/blob/c3b913cda2467e4d2db40535395f1282fb6d7ebc/src/Sylius/Bundle/AdminBundle/Controller/CustomerStatisticsController.php#L41-L47)

```php
        $customer = $this->customerRepository->find($customerId);
        if (null === $customer) {
            throw new HttpException(
                Response::HTTP_BAD_REQUEST,
                sprintf('Customer with id %s doesn\'t exist.', (string) $customerId),
            );
        }
```
Sorry, missed it at first review.